js/message_test.js

```diff
@@ -1049,13 +1049,7 @@ describe('Message test suite', function() {
     var nested = new proto.jspb.test.Deeply.Nested.Message();
     nested.setCount(5);
     msg.setDeeplyNestedMessage(nested);
-
-    // After a serialization-deserialization round trip we should get back the
```

Comment:
I found that we don't currently generate `fromObject` methods (the code for it exists in js_generator.cc but is never called), so it seemed simplest to just avoid doing the round trip.